### PR TITLE
DOCSP-46067 Compass 1.45.0 RN

### PR DESCRIPTION
**Pull request by ajhuh-mdb**
## DESCRIPTION
- Compass 1.45.0 release notes

## STAGING
https://deploy-preview-706--docs-compass.netlify.app/release-notes/#compass-1.45.0

## JIRA
https://jira.mongodb.org/browse/DOCSP-46067

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/release-notes.txt [hunk 1]
```diff
@@ -10,6 +10,19 @@ Release Notes
...
 
+|compass| 1.45.0
+----------------
+
+*Released December 4, 2024*
+
+Bug Fixes:
+
+- Disables UTF8 validation when retrieving Compass metadata 
+  (:issue:`COMPASS08517`)
```
Comment (jmd-mongo) [inline source/release-notes.txt:21]: [typo]

I think this ought to be `COMPASS-8517`.

**Review by jmd-mongo — CHANGES_REQUESTED** [review]
Thanks for these release notes, @ajhuh-mdb! I noted a typo.

Thanks,
Joe

File: source/release-notes.txt (hunk 1 above)
Comment (ajhuh-mdb) [reply source/release-notes.txt:21]: Ack sorry about missing that 😓 Updating.

**Comment by ajhuh-mdb** [comment]
Sorry for missing that typo @jmd-mongo. This should be ready for another look!

**Review by jmd-mongo — APPROVED** [review]
Thanks, @ajhuh-mdb! This one LGTM at this point.

Thanks,
Joe

**Comment by betsybutton** [comment]
LGTM!

**Review by betsybutton — APPROVED** [review]

**Pull request merged**